.travis.yml

```diff
@@ -120,11 +119,11 @@ install:
         conda install --yes pyopengl scipy numpy$NUMPY networkx;
         pip install -q numpydoc PySDL2;
         if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+          conda install --yes wxpython matplotlib jupyter pyqt=4 pillow decorator six;
           if [ "${PYTHON}" == "3.6" ]; then
-            pip install -q freetype-py husl pypng cassowary pillow decorator six;
+            pip install -q freetype-py husl pypng cassowary imageio;
```

Comment:
I'm surprised this works, some other dependencies must pull in `six`, `pillow`, and `decorator`, then?

Reply:
Don't miss that I just moved those dependencies to the conda install.

Reply:
Oh yes I did, thanks